eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -35,7 +67,11 @@ function Get-javascript-DocsMsTocData($packageMetadata, $docRepoLocation) {
     $readmeMetadata = &$GetDocsMsMetadataForPackageFn -PackageInfo $packageMetadata.FileMetadata
     $packageLevelReadmeName = $readmeMetadata.DocsMsReadMeName
   }
+  return $packageLevelReadmeName
+}
 
+function Get-javascript-DocsMsTocData($packageMetadata, $docRepoLocation) {
+  $packageLevelReadmeName = Get-javascript-PackageLevelReadme -packageMetadata $packageMetadata
```

Comment:
Similar comment as this https://github.com/Azure/azure-sdk-for-java/pull/29818#discussion_r917525691 

If this is the only place it's used, we may not need to define the generic function name in `common.ps1`

Reply:
This `Get-javascript-PackageLevelReadme` fn is being called in Service-level-readme-automation to get the right package readme name.

Reply:
Talked offline fixed by using helper